SequenceAnalysis/src/org/labkey/sequenceanalysis/analysis/PicardAlignmentMetricsHandler.java

```diff
@@ -191,10 +191,10 @@ public void processFilesRemote(List<SequenceOutputFile> inputFiles, JobContext c
             RecordedAction action = new RecordedAction(getName());
             action.setStartTime(new Date());
 
-            boolean collectSummary = params.containsKey("collectSummary") && params.optBoolean("collectSummary", false);
-            boolean collectInsertSize = params.containsKey("collectInsertSize") && params.optBoolean("collectInsertSize", false);
-            boolean collectWgs = params.containsKey("collectWgs") && params.optBoolean("collectWgs", false);
-            boolean collectWgsNonZero = params.containsKey("collectWgsNonZero") && params.optBoolean("collectWgsNonZero", false);
+            boolean collectSummary = params.has("collectSummary") && params.optBoolean("collectSummary", false);
+            boolean collectInsertSize = params.has("collectInsertSize") && params.optBoolean("collectInsertSize", false);
+            boolean collectWgs = params.has("collectWgs") && params.optBoolean("collectWgs", false);
+            boolean collectWgsNonZero = params.has("collectWgsNonZero") && params.optBoolean("collectWgsNonZero", false);
```

Comment:
I believe these four lines can each be simplified to `params.optBoolean("collectWgsNonZero")` and equivalent.

Reply:
Just FYI